tools/tests/apps/modules/.meteor/release

```diff
@@ -1 +1 @@
-METEOR@1.8
+METEOR@1.8.1
```

Comment:
Shouldn't this be 1.9-alpha.0?

Reply:
The modules test app runs from a checkout during Circle CI self-tests, so the `.meteor/release` file doesn't really matter (although we totally could update the version on `release-1.9`). The reason it shows up in the changes for this PR is because I recently merged `release-1.8.2` into `release-1.9` (and both PRs are targeting `master`, which doesn't have the change).